src/ConfigEditor.tsx

```diff
@@ -14,8 +14,11 @@ export class ConfigEditor extends PureComponent<Props, State> {
       onOptionsChange({...options, jsonData: {...options.jsonData, enableImpersonation: event.target.checked}})
     }
     const onTokenChange = (event: ChangeEvent<HTMLInputElement>) => {
-      onOptionsChange({...options, jsonData: {...options.jsonData, accessToken: event.target.value}})
+      onOptionsChange({...options, secureJsonData: {...options.secureJsonData, accessToken: event.target.value}})
```

Comment:
It is not clear to me what makes it actually secure? Is this just a name change?

Reply:
It's supposed to store the access token in a different structure - `secureJsonData` instead of `jsonData`.

Reply:
I've been following https://grafana.com/developers/plugin-tools/how-to-guides/data-source-plugins/add-authentication-for-data-source-plugins#store-configuration-in-securejsondata

Reply:
I see - now it makes sense!